### PR TITLE
add video dialog component for custom landing page (fixes #7175)

### LINKED ISSUE #7175: Landing: Video not playing
Video in the landing(hero) component does not play, the button is not linked to a dialog box

### PR DESCRIPTION
**Pull request by Mutugiii**
Allows landing video to play.

Fixes #7175

### CODE REVIEW

File: src/app/landing/landing-hero/landing-hero.component.ts [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-  @Output() playVideoEvent = new EventEmitter<boolean>();
+  constructor(
+    private dialog: MatDialog
+  ) {}
 
   playVideo() {
-    this.playVideoEvent.emit(true);
+    this.dialog.open(DialogsVideoComponent, {
+      data: {
+        videoUrl: 'assets/landing-page/video/landing.mp4'
```
Comment (lmmrssa) [inline src/app/landing/landing-hero/landing-hero.component.ts:18]: Right now I do not see this PR improving any implementation. @dogi what is the plan for this reason to be removed from image?

Comment (Mutugiii) [reply src/app/landing/landing-hero/landing-hero.component.ts:18]: In the current state the emitted event is not being used anywhere and the video is not playing on click.

Comment (lmmrssa) [reply src/app/landing/landing-hero/landing-hero.component.ts:18]: Need to test both container and non container case due to relative path. So please test with different language, before merging.

**Review by lmmrssa — APPROVED** [review]
Please test before merging

**Review by dogi — APPROVED** [review]

**Pull request merged**